main/jlp/net/http_api.cpp

```
@@ -805,6 +808,77 @@ esp_err_t hello_get(httpd_req_t* req) {
+    char val[80];
+    if (httpd_query_key_value(q, "stop", val, sizeof(val)) == ESP_OK) {
+      // Never yank a widget-owned client: the widget's own teardown must
+      // drop its pixel references first or LVGL keeps a freed buffer.
+      if (!stream_client_stop_diagnostic()) {
+        ok = false;
+        err = "stream is owned by the layout's widget";
+      }
+    } else if (httpd_query_key_value(q, "start", val, sizeof(val)) == ESP_OK) {
+      std::string host = sk_server().host;
+      uint16_t soak_port = 5004;
+      if (httpd_query_key_value(q, "host", val, sizeof(val)) == ESP_OK) host = val;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)